setup.cfg

```diff
@@ -34,7 +34,7 @@ package_dir =
 packages = find:
 install_requires =
     django>=2.2,<3.3
-    PyJWT>=1.4.2,<2.0
+    PyJWT>=1.4.2
```

Comment:
```suggestion
    PyJWT>=2.4.0
```

Since the vulnerability is only patched in versions 2.4.0 and above, let's be explicit to make sure that we get that version